src/cpu/aarch64/xbyak_aarch64/xbyak_aarch64/xbyak_aarch64_code_array.h

```diff
@@ -30,7 +30,7 @@ inline void *AlignedMalloc(size_t size, size_t alignment) {
 }
 
 inline void AlignedFree(void *p) {
-#ifdef _MSC_VER
+#if defined(_MSC_VER) || defined(__MINGW64__)
```

Comment:
I don't have any knowledge about these APIs, but if this is a windows thing, would it be better/clearer to _just_ use a windows specific ifdef rather than specify each windows compiler?

Reply:
I agree. WIN32 should be enough here.